World_Simulation/README.md

```diff
@@ -0,0 +1,16 @@
+# World Simulation
+
+
+Purpose: see if GeoClaw could handle a storm with the topography of world.
+
+
+Topography was selected from NOAA Grid Extract. 12 topography files from -180 to 180 were extracted and later adjusted to be from 0 to 360. Longitude of topography files was from -60 to 60.
```

Comment:
We need to be much more explicit as to how you got this data, where it came from, and exactly what you did to transform it if you are not going to provide a script to automatically fetch it.

Reply:
Hi Dr. Mandli. I added code that I used to transform the data. I will also look into making the script automatically fetch the topography files.